### PR TITLE
chapter 4

### PR DESCRIPTION
**Pull request by starocketeer**

### CODE REVIEW

**Comment by starocketeer** [comment]
close pull request

**Comment by starocketeer** [comment]
open pull request

**Comment by starocketeer** [comment]
Ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss]: remove empy file you not use

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class StaticPagesController < ApplicationController
+    def home; end
```
Comment (ngocvt-0484) [inline app/controllers/static_pages_controller.rb:2]: tab 2 space please

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "static_pages#home"
+  
+  get   "/home", to: "static_pages#home"
```
Comment (ngocvt-0484) [inline config/routes.rb]:
```suggestion
  get "/home", to: "static_pages#home"       
```

File: app/controllers/static_pages_controller.rb (hunk 2 above)
Comment (starocketeer) [reply app/controllers/static_pages_controller.rb:2]: done

File: app/assets/stylesheets/users.scss (hunk 1 above)
Comment (starocketeer) [reply app/assets/stylesheets/users.scss]: done

File: config/routes.rb (hunk 3 above)
Comment (starocketeer) [reply config/routes.rb]: done

**Comment by starocketeer** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+    def new; end
+end
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]:
```suggestion
class UsersController < ApplicationController
  def new; end
end

```

File: app/helpers/users_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
+end
```
Comment (saobv-2223) [inline app/helpers/users_helper.rb]: remove file that never used

File: app/controllers/users_controller.rb (hunk 4 above)
Comment (starocketeer) [reply app/controllers/users_controller.rb]: done

File: app/helpers/users_helper.rb (hunk 5 above)
Comment (starocketeer) [reply app/helpers/users_helper.rb]: done

**Comment by starocketeer** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def new; end
+end
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]:
```suggestion
end

```

**Review by saobv-2223 — COMMENTED** [review]
Resolve all the conflict files, please

File: app/controllers/users_controller.rb (hunk 6 above)
Comment (starocketeer) [reply app/controllers/users_controller.rb]: done

**Comment by starocketeer** [comment]
Ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**